apps/et/perftest/base/kustomization.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
```

Comment:
this file will need referencing in https://github.com/hmcts/cnp-flux-config/blob/master/clusters/perftest/base/kustomization.yaml

Reply:
@timja - Hi Tim, would my latest commit work? Thanks

Reply:
@timja - Hi Tim, gentle nudge to review my latest commit/comment? Thanks

Reply:
Yes that's good, if you request my review again i get a list of PRs to follow up on :)

![image](https://user-images.githubusercontent.com/21194782/174782466-d577be51-5450-433f-bbda-999114ad4b3d.png)

hit the cycling arrow